examples/README

```diff
@@ -1 +1,84 @@
-Here are two sample programs which use QTermWidget for displaying a terminal
+Examples using QTermWidget
+==========================
+
+This directory contains minimal examples showing how to embed and use `QTermWidget` in both C++ and Python (PyQt/PySide) applications.
+
+Contents
+--------
+* `cpp/main.cpp` – Basic single–terminal C++ example with menu actions (Find / Copy / Paste / About Qt) and runtime selection of color schemes and key bindings via command‑line arguments.
+* `cpp/RemoteTerm/` – A simple remote client example that connects to a TCP server exposing a pseudo terminal. Local keystrokes are sent to the remote host; data received over the socket is written into the local PTY backing the widget.
+* `pyqt/` – Python examples (PyQt/PySide) demonstrating how to instantiate and configure the terminal widget from Python. See the files inside for details.
+
+Building C++ Examples
+---------------------
+
+Configure the build (top level) and enable example targets as needed:
+
+```
+mkdir -p build
+cd build
+cmake -DBUILD_EXAMPLE=ON -DBUILD_EXAMPLE_REMOTETERM=ON ..
+cmake --build . -j
+```
+
+Resulting binaries (paths relative to the build directory):
+* `./example` – the basic example.
+* `./remoteterm` – the remote terminal client (only if `BUILD_EXAMPLE_REMOTETERM=ON`).
+
+Running the Basic Example
+-------------------------
+```
+./example [colorScheme] [keyBinding]
+```
+
+You can pass any available color scheme or key binding name (both optional). To list what is available, just run without arguments – they are also printed to stdout on startup.
+
+RemoteTerm Example
+------------------
+Usage:
+```
+./remoteterm <ipaddr> <port>
+```
+
+It will:
+1. Start a local, empty PTY via `startTerminalTeletype()`.
+2. Connect a `QTcpSocket` to the remote server.
+3. Forward local user keystrokes (via the `sendData` signal) to the socket.
+4. Write any incoming socket data directly into the PTY so it appears in the widget.
+
+This is a minimal client-only demonstration. A corresponding server program (not included here) must:
+* Accept a TCP connection.
+* Spawn or attach to a PTY on the server side.
+* Forward data bidirectionally between the PTY and the socket.
+
+Python Examples
+---------------
+Install the Python bindings (PyQt6 recommended for Qt6 builds) and run the scripts directly, for example:
```

Comment:
Minor: PyQt5 is no longer supported. Only PyQt6 is supported, so it is not only recommended, but also needed for the Python example.